README.md

```diff
@@ -84,14 +84,14 @@ Ensure the properties there fit your environment.
 
 #### Build commands:
 
-- Without tests: `mvn clean package -DskipTests `
+- Without tests: `mvn clean package -Dskip.unit.tests`
```

Comment:
1. skipTests property is used for surefire plugin too, generic
2. skipTests was used only for unit tests, but from name it could confusing that it controls all tests
3. new property is used for exact test activation in code coverage profile too.